src/scss/includes/menu.scss

```diff
@@ -0,0 +1,116 @@
+.menu__overlay {
+  display: none;
+  height: 100vh;
```

Comment:
There is a regression on Chrome Android, that might be due to these 100vh values, here and below.  
After opening and closing the burger menu, the page can be scrolled, and a blank zone is visible at the bottom of the screen:
<img src="https://user-images.githubusercontent.com/4726554/52644779-be7da200-2edf-11e9-9a96-351d3e591f38.png" width="400">